.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Thanks for submitting a pull request! Before pressing that tempting green submit
+button, please make sure you check off all the following boxes:
+
+- [ ] All existing unit tests pass in all supported Python versions (2.6, 2.7,
+      and 3.3+)
+- [ ] New tests have been added to verify the new functionality / bug fix
+- [ ] All code works on at least Windows, Mac OS X, and Linux
```

Comment:
Yeah, and in line with your other comment it isn't something we often remind people on.